code/game/machinery/tcomms/nttc.dm

```diff
@@ -40,18 +40,30 @@
 		"Maintenance Technician" = "engradio",
 		"Station Engineer" = "engradio",
 		// Central Command
-		"Emergency Response Team Engineer" = "dsquadradio", // I know this says deathsquad but the class for responseteam is neon green. No.
+		"Custodian" = "dsquadradio", // I know this says deathsquad but the class for responseteam is neon green. No.
```

Comment:
Damn even the CC jannies get fancy radios?
For real though, not sure why this rank exists as it really is not even used.

Reply:
I genuinely have no idea, it is present in lines 81 and 391 of access.dm, and in both instances it is explicitly mentioned as a "CentComm" job.

There is no mention of it elsewhere, and the only gear preset I can see it belonging to is "Tournament Janitor" in line 972 of outfit_admin.dm, which seems to be related to the Thunderdome in some way or another.

I just copied the entirety of the jobs listed in access.dm so that I wouldn't break any existing thing, but I can remove it if you think it might cause issues.

Alternatively, I could have just swapped out the names in access.dm, but since I was unsure if it would make any sense to rename "custodian" to "tournament custodian", I left it as is. Because what if it is actually "shuttle custodian", the very same that cuts his guts and delays the launch by 2 minutes when Cult uses a shuttle curse?

Reply:
I dont think it's a problem to keep it, I was more highlighting the oddity of it more than anything

Reply:
I was flummoxed myself with its presence, which is why I did so much digging into possible explanations.

None ever really stuck fully, so I just resigned myself to the technical debt, maybe someday someone will make a proper Thunderdome mode (think like TG minigames to entertain ghosts), in which case having this rank around would be useful.

Resolving the conversation.